kebechet/managers/version/release_triggers.py

```diff
@@ -72,20 +102,16 @@ def adjust_version_file(self, file_path: str) -> Optional[tuple]:
                             environment_details=ManagerBase.get_environment_details(),
                         )
                     ) from e
-
                 _LOGGER.info("Computed new version: %s", new_version)
-
                 content[idx] = f'__version__ = "{new_version}"'
                 changed = True
-
-        if not changed:
-            return None
-
-        # Apply changes.
-        with open(file_path, "w") as output_file:
-            output_file.write("\n".join(content))
-            # Add new line at the of file explicitly.
-            output_file.write("\n")
+            if not changed:
+                return None
+            # Apply changes.
+            with open(file_path, "w") as output_file:
+                output_file.write("\n".join(content))
+                # Add new line at the of file explicitly.
+                output_file.write("\n")
```

Comment:
This part should be outside the for loop, right?
or else as soon as first line isn't __version__ it would return None.
we should trace to the whole loop before we check for changed.

Reply:
My bad, while refactoring, I must have tab- pushed it . Thanks for the review.